server/core/admin.cc

```diff
@@ -938,6 +971,7 @@ bool mxs_admin_init()
                                             MHD_OPTION_SOCK_ADDR, &addr,
                                             !this_unit.using_ssl ? MHD_OPTION_END :
                                             MHD_OPTION_HTTPS_MEM_KEY, this_unit.ssl_key.c_str(),
+                                            MHD_OPTION_HTTPS_PRIORITIES, this_unit.ssl_cipher.c_str(),
```

Comment:
For the sake of consistency, I'd say add this after `MHD_OPTION_HTTPS_MEM_CERT`. That way the certificate arguments follow each other.